msg/json/en.json

```diff
@@ -291,7 +291,7 @@
 	"LISTS_ISEMPTY_TITLE": "%1 is empty",
 	"LISTS_ISEMPTY_TOOLTIP": "Returns true if the list is empty.",
 	"LISTS_INLIST": "in list",
-	"LISTS_INDEX_OF_HELPURL": "https://github.com/google/blockly/wiki/Lists#finding-items-in-a-list",
+	"LISTS_INDEX_OF_HELPURL": "https://github.com/google/blockly/wiki/Lists#getting-items-from-a-list",
```

Comment:
I think the previous version of this is more correct, so it's possible this entire file should just be reverted as this is somehow putting back an old change (hence updating the date backwards and using a worse link)

Reply:
Done